traffic_ops/v4-client/server_server_capabilities.go

```diff
@@ -39,7 +39,9 @@ func (to *Session) DeleteServerServerCapability(serverID int, serverCapability s
 	if opts.QueryParameters == nil {
 		opts.QueryParameters = url.Values{}
 	}
-	opts.QueryParameters.Set("serverId", strconv.Itoa(serverID))
+	if serverID != 0 {
+		opts.QueryParameters.Set("serverId", strconv.Itoa(serverID))
+	}
```

Comment:
This method doesn't work without specifying a server ID - why do this?

Reply:
reverted the client method changes.